### ISSUE TITLE
SRT: 拉流播放不會呼叫 on_play 的 http hook 

### ISSUE THREAD

**Issue by jeantean** [ossrs/srs#2278]
**描述(Description)**

> 描述你遇到了什么问题(Please description your issue here)
當用 ffmpeg 推 SRT 流到 SRS 4.0.87 時可以有
http: on_publish ok, client_id=30x8c79z,
http: on_unpublish ok, client_id=1fb07226
但是 ffplay 拉 SRT 流時
不會去呼叫
http: on_play 
rtmp 的沒有問題可以正常呼叫 http: on_play 

1. SRS版本(Version): `4.0.87`
1. SRS的日志如下(Log):
```

[2021-04-08 00:44:36.251][Trace][1120][30x8c79z] http: on_publish ok, client_id=30x8c79z, url=http://auth.amib.tv/srt-auth.php, request={"action":"on_publish","client_id":"30x8c79z","ip":"127.0.0.1",
"vhost":"__defaultVhost__","app":"live","tcUrl":"rtmp://127.0.0.1:17935/live","stream":"livestream","param":"?vhost=127.0.0.1"}, response=0
[2021-04-08 00:44:32.780][Trace][1120][1fb07226] http: on_unpublish ok, client_id=1fb07226, url=http://auth.amib.tv/srt-auth.php, request={"action":"on_unpublish","client_id":"1fb07226","ip":"127.0.0
.1","vhost":"__defaultVhost__","app":"live","stream":"livestream","param":"?vhost=127.0.0.1"}, response=0


```
1. SRS的配置如下(Config):
```

listen              17935;
max_connections     1000;
daemon              on;
srs_log_tank        file;
srs_log_file        ./objs/srs.log;


srt_server {
    enabled on;
    listen 10081;
}
vhost __defaultVhost__ {

  http_hooks {
        enabled         on;
        on_connect      http://auth.amib.tv/srt-auth.php;
        on_close        http://auth.amib.tv/srt-auth.php;
        on_publish      http://auth.amib.tv/srt-auth.php;
        on_unpublish    http://auth.amib.tv/srt-auth.php;
        on_play         http://auth.amib.tv/srt-auth.php;
        on_stop         http://auth.amib.tv/srt-auth.php;

    }



}

```

**重现(Replay)**

> 重现Bug的步骤(How to replay bug?)

1. `ffplay srt://127.0.0.1:10081?streamid=#!::h=live/livestream,m=request
1. `xxxxxx`
1. `xxxxxx`

**期望行为(Expect)**

> 描述你期望发生的事情(Please describe your expectation)
期望可以在 SRT 拉流(request) 時在 on_play 加上與 rtmp 一樣的 hook 功能

**Comment by allanlei** [comment]
Same for `4.0.126`

**Comment by runner365** [comment]
@jeantean Hello, currently the SRT streaming does not have any linkage with the RTMP HTTP hook, as it is an internal SRT streaming.

This is not supported temporarily.

`TRANS_BY_GPT3`

**Comment by winlinvip** [comment]
Fixed in SRS 5.0
